node/utils/helpers.ts

```diff
@@ -39,19 +39,38 @@ export const isMessageValid = (message: string) => {
   )
 }
 
-export const formatMessage = (message: string, data: any, type = 'order') => {
+export const formatMessage = (
+  message: string,
+  data: any,
+  type = 'order',
+  addOneDayToDeliveryDate = false
+) => {
   let newMessage = message
 
   const { creationDate } = data
   const estimateDays = data.shippingData?.logisticsInfo?.[0]?.shippingEstimate
-  const estimateDate = data.shippingData?.logisticsInfo?.[0]?.shippingEstimateDate
+  const estimateDate =
+    data.shippingData?.logisticsInfo?.[0]?.shippingEstimateDate
+
+  const deliveryDate = estimateDate
+    ? new Date(estimateDate)
+    : new Date(creationDate)
 
-  let deliveryDate = estimateDate ? new Date(estimateDate) : new Date(creationDate)
   if (!estimateDate) {
-    deliveryDate.setDate(deliveryDate.getDate() + Number(String(estimateDays)?.replace(/\D/g, '')))
+    deliveryDate.setDate(
+      deliveryDate.getDate() + Number(String(estimateDays)?.replace(/\D/g, ''))
```

Comment:
Teoretic asta nu e chiar corecta, dar daca asa era inainte, poate ramane asa, pentru ca '2bd' ptr o comanda plasata vineri ar trebui sa fie marti de ex, daca curierul nu livreaza in weekend.